features/support/sharedSteps.js

```diff
@@ -45,24 +46,36 @@ Then(
   }
 );
 
+// eslint-disable-next-line prefer-arrow-callback
```

Comment:
It might be cleaner to disable the `prefer-arrow-callback` and `no-underscore-dangle` linting rules for the entire file, since they both come up more than once, and probably will continue to come up if we add more steps.

See https://eslint.org/docs/2.13.1/user-guide/configuring#disabling-rules-with-inline-comments